src/containers/Ledgers/Legend.tsx

```diff
@@ -0,0 +1,71 @@
+import { useTranslation } from 'react-i18next'
+import {
+  TransactionAction,
+  TransactionCategory,
+} from '../shared/components/Transaction/types'
+import { useLocalStorage } from '../shared/hooks'
+import { TransactionActionIcon } from '../shared/components/TransactionActionIcon/TransactionActionIcon'
+import './css/legend.scss'
+
+export const LEGEND_STORAGE_KEY = 'explorer-hide-legend'
+
+export const Legend = () => {
+  const { t } = useTranslation()
+  const [hidden, setHidden] = useLocalStorage<boolean>(
+    LEGEND_STORAGE_KEY,
+    false,
+  )
+
+  const actions = [
+    TransactionAction.CREATE,
+    TransactionAction.MODIFY,
+    TransactionAction.FINISH,
+    TransactionAction.CANCEL,
+    TransactionAction.SEND,
+  ]
+
+  const categories = [
+    TransactionCategory.PAYMENT,
+    TransactionCategory.DEX,
+    TransactionCategory.NFT,
+    TransactionCategory.ACCOUNT,
+    TransactionCategory.PSEUDO,
+    TransactionCategory.UNKNOWN,
+  ]
```

Comment:
Can't you just use the enum itself instead of needing to list it out again?

Reply:
This is a subset of all enums and a special order.